test/env/client/postgres.go

```diff
@@ -13,15 +14,16 @@ type Postgres struct {
 }
 
 func New(address, user, password, dbName string) (*Postgres, error) {
-	url := fmt.Sprintf("postgres://%s:%s@%s/%s", user, password, address, dbName)
+	url := fmt.Sprintf("postgres://%s:%s@%s/%s?sslmode=disable", user, password, address, dbName)
```

Comment:
since the mode at server side is able to handle both ssl and not ssl trafic (ssl mode: prefered) we don't see a point in passing here TLS config since all we need to check here is persistence of data, TLS i verified in integration test with other components like keystone and command